profile/README.md

```diff
@@ -0,0 +1,22 @@
+<p align="center">
+  <img src="https://avatars.githubusercontent.com/u/871554?s=200&v=4" alt="Betterment's profile image"/>
+</p>
+
+[![Hiring badge](https://img.shields.io/badge/We're-hiring-green.svg)](https://www.betterment.com/careers/)
+
+We're passionate about building the simplest, most sophisticated investment platform on the planet. Check out our [engineering blog](https://www.betterment.com/category/engineering/)
+
+### Upcoming talks and events
+
+@yieldjessyield and @caffeineflo speaking at [droidcon london](https://www.london.droidcon.com/program/is-flutter-the-coke-zero-of-the-mobile-world%3F)
+
+### Talks our engineers have given
```

Comment:
we could have this say something like "Hear from our engineers!" and then include stuff like the podcast @samandmoore did

Reply:
Done!